Dockerfile

```diff
@@ -231,8 +235,7 @@ ARG CONTAINERD_VERSION
 RUN --mount=from=containerd-src,src=/usr/src/containerd,rw \
     --mount=target=/root/.cache,type=cache <<EOT
   set -ex
-  git fetch origin
-  git checkout -q "$CONTAINERD_VERSION"
+  git switch -q -d "$CONTAINERD_VERSION"
```

Comment:
Maybe avoid these new commands if they are not really needed for better readability

Reply:
Fixed to not use git switch.